src/linux/network.rs

```diff
@@ -205,50 +205,50 @@ impl NetworkData {
 
 impl NetworkExt for NetworkData {
     fn get_received(&self) -> u64 {
-        self.rx_bytes as u64 - self.old_rx_bytes as u64
+        self.rx_bytes - self.old_rx_bytes
     }
 
     fn get_total_received(&self) -> u64 {
-        self.rx_bytes as u64
+        self.rx_bytes 
     }
 
     fn get_transmitted(&self) -> u64 {
-        self.tx_bytes as u64 - self.old_tx_bytes as u64
+        self.tx_bytes  - self.old_tx_bytes 
```

Comment:
You added one extra whitespace here and in some other places. Could you remove them please? :)

Reply:
Hi, I removed them.
I pushed the correction.